app/src/main/java/co/tinode/tindroid/db/MessageDb.java

```diff
@@ -218,13 +221,49 @@ public static Cursor query(SQLiteDatabase db, long topicId, int pageCount, int p
                     " AND "
                         + COLUMN_NAME_STATUS + "<=" + BaseDb.STATUS_VISIBLE +
                 " ORDER BY "
-                    + COLUMN_NAME_TS + " DESC, "
                     + COLUMN_NAME_SEQ + " DESC" +
                 " LIMIT " + (pageCount * pageSize);
 
         return db.rawQuery(sql, null);
     }
 
+    /**
+     * Query for the latest messages. The query returns NULL-filled rows whenever there is a gap in seq values.
+     * See explanation here: https://stackoverflow.com/questions/31589843/how-do-i-find-gap-in-sqlite-table
+     *
+     * @param db        database to select from;
+     * @param topicId   Tinode topic ID (topics._id) to select from
+     * @param maxSeq    Maximum known seqId in the table.
+     * @param pageCount number of pages to return
+     * @param pageSize  number of messages per page
+     * @return cursor with the messages.
+     */
+    public static Cursor query(SQLiteDatabase db, long topicId, int maxSeq, int pageCount, int pageSize) {
+        final String sql ="SELECT " +
```

Comment:
Add a space between the equals and the double quote in `="SELECT` please.

Reply:
Done